recipes/qcs-api-client-common/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "qcs-api-client-common" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: qcs-api-client-common-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/rigetti/qcs-api-client-rust/archive/refs/tags/common/v{{ version }}.tar.gz
+  sha256: ee84c504bd5843ad8f129b50594ab9afaaa8bfa32d8f43a016b4ecc5bce1ae37
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - maturin >=1.0.0,<2.0.0
+    - cargo-bundle-licenses
+    - python
+  host:
+    - python
+    - toml
+
+outputs:
+  - name: qcs_api_client_common
+    build:
+      run_exports:
+        - {{ pin_subpackage("qcs-api-client-common", max_pin="x.x") }}
+    requirements:
+      host:
+        - {{ pin_subpackage("qcs-api-client-common") }}
+      run:
+        - {{ pin_subpackage("qcs-api-client-common") }}
+    test:
+      imports:
+        - qcs_api_client_common
+      requires:
+        - pip
+      commands:
+        - pip check
+
+  - name: qcs-api-client-common
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage("qcs-api-client-common", max_pin="x.x") }}
```

Comment:
What's the reason for these run_exports? We normally only have them on shared libraries but the packages here look like Python packages.

Reply:
Silly copy-paste from `qcs-sdk-python`. I'll need to correct that feedstock afterwards